src/pages/languages.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * @emails react-core
+ * @flow
+ */
+
+import Layout from 'components/Layout';
+import Container from 'components/Container';
+import Header from 'components/Header';
+import TitleAndMetaTags from 'components/TitleAndMetaTags';
+import React from 'react';
+import {media, sharedStyles} from 'theme';
+
+// $FlowFixMe This is a valid path
+import languages from '../../content/languages.yml';
+
+const completedLangauges = languages
+  .filter(language => language.is_completed)
+  .sort((a, b) => a.code.localeCompare(b.code));
+const inProgressLangauges = languages
+  .filter(language => !language.is_completed)
+  .sort((a, b) => a.code.localeCompare(b.code));
+
```

Comment:
I think this doesn't affect any performance, but I'll want to comment it and maybe can improve something. Instead of iterating the whole array of languages twice:

```js
const [ completedLanguages, inProgressLanguages ] = languages.reduce((memo, language) => {
  if (language.is_completed) {
    memo[0].push(language);
  } else {
    memo[1].push(language);
  }
  return memo;
}, [ [], [] ])
  .map(subArray => subArray.sort((a, b) => a.code.localeCompare(b.code)));
```

I hope it helps, by the way, there's a typo in variable names: `completedLanguages` and `inProgressLanguages`.